.github/workflows/CI.yml

```diff
@@ -15,7 +15,10 @@ jobs:
       - run: npm install
 
       # Run tests
-      - run: npm run test-all
+      - run: |
+          echo adone >> node_modules/@definitelytyped/dtslint-runner/expectedFailures.txt
+          sed --in-place 's%return undefined;%//&%' node_modules/@definitelytyped/dtslint-runner/dist/main.js
```

Comment:
can you make this change to @definitelytyped/dtslint-runner instead of patching it here?

Reply:
Will do!